rustls/src/server/hs.rs

```diff
@@ -341,6 +341,9 @@ impl ExpectClientHello {
             certkey.get_key().algorithm(),
         );
 
+        // TODO we can avoid over-allocating here by removing non-suitable
+        // suites from `suitable_suites` above
```

Comment:
This was more a comment for rustls maintainers. I can of course make the change, but it would touch parts of the code base not related to ffdhe.

I can also remove the comment.

Reply:
I am OK with this